crates/holochain/CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+- Performance improvement when sending validation receipts. When a batch of DHT ops is being processed and an author is unreachable while attempting to send a validation receipt, all the other receipts for that
+  author will be skipped. The network send already has a retry built in so when an author was offline and a batch of `N` receipts were ready to be sent it would take `N` minutes for the workflow to work through
+  them. This was unfair to other nodes that are still online because they would have to wait unreasonably long to get their validation receipts.
```

Comment:
Validation receipts are pretty small, right? This seems like a lot of work to implement pseudo-batching, would it be better to just actually batch them?

Reply:
Yeah it's a hash, a list of validators and a signature - nothing big to be sent. This is a good point and something we may as well look at while making changes to how they're sent anyway. It would simplify the logic a bit in the workflow too

Reply:
Done!